test/tests/providers/global/prometheus-agent.rules.test.yml

```diff
@@ -13,7 +13,7 @@ tests:
       - alertname: PrometheusAgentFailing
         eval_time: 10m
       - alertname: PrometheusAgentFailing
-        eval_time: 25m
+        eval_time: 25ms
```

Comment:
```suggestion
        eval_time: 25m
```

Reply:
Oh did I make a typo with the old ctrl+s :D